apps/src/dance/Dance.js

```diff
@@ -176,16 +179,26 @@ Dance.prototype.init = function (config) {
 
   ReactDOM.render(
     <Provider store={getStore()}>
-      <AppView
-        visualizationColumn={
-          <DanceVisualizationColumn
-            showFinishButton={showFinishButton}
-            setSong={this.setSongCallback.bind(this)}
-            resetProgram={this.reset.bind(this)}
-          />
+      <ErrorBoundary
+        fallback={<ErrorFallbackPage />}
+        onError={(error, componentStack) =>
+          Lab2MetricsReporter.logError('Dance Party Error', error, {
```

Comment:
Maybe `"Uncaught Dance Party React error"`? Or we can even stick with `"Uncaught React error"` once we have the common properties set up, as we'll have the `appName: 'Dance'` on the payload that we can filter by.

Reply:
I flipped it back to the old error and put in the common properties.